VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -211,24 +243,22 @@ public void onClick(View v) {
 
             @Override
             public void onClick(View v) {
-            environment.getRouter().forceLogout(getActivity(), environment.getSegment(), environment.getNotificationDelegate());
+                environment.getRouter().forceLogout(getActivity(), environment.getSegment(), environment.getNotificationDelegate());
             }
         });
 
 
-        
-
         TextView version_tv = (TextView) layout.findViewById(R.id.tv_version_no);
-        try{
+        try {
             String versionName = PropertyUtil.getManifestVersionName(getActivity());
 
-            if(versionName != null) {
+            if (versionName != null) {
                 String envDisplayName = environment.getConfig().getEnvironmentDisplayName();
                 String text = String.format("%s %s %s",
                         getString(R.string.label_version), versionName, envDisplayName);
                 version_tv.setText(text);
             }
-        }catch(Exception e) {
+        } catch (Exception e) {
```

Comment:
Again, we can remove the generic catch block; the `PropertyUtil.getManifestVersionName()` method has one anyway.